docs/source/staking/running_a_worker.rst

```diff
@@ -131,25 +135,24 @@ Initialize a new Worker
 
 .. code:: bash
 
-    docker run -it --rm \
-    --name ursula       \
+    docker run -it --rm  \
+    --name ursula        \
     -v ~/.local/share/nucypher:/root/.local/share/nucypher \
-    -v ~/.ethereum/:/root/.ethereum \
-    -p 9151:9151                    \
-    -e NUCYPHER_KEYRING_PASSWORD    \
-    nucypher/nucypher:latest        \
-    nucypher ursula init            \
-    --provider <PROVIDER URI>       \
-    --network <NETWORK NAME>        \
-    --signer <SIGNER URI>           \
+    -v ~/.ethereum/:/root/.ethereum               \
+    -p 9151:9151                                  \
+    -e NUCYPHER_KEYRING_PASSWORD                  \
+    nucypher/nucypher:latest                      \
+    nucypher ursula init                          \
+    --signer keystore:///root/.ethereum/keystore  \
```

Comment:
Not for this PR, but would it make sense to make that a default option?

Reply:
It varies with the OS and with the ethereum network, so I wouldn't introduce a default

Reply:
Both the OS and the network are know at the moment of running this command, so theoretically we could look into the default keystore path and see if there's actually a keystore in it.